doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -58,6 +58,7 @@ Please make sure to set your preferred storage location ***before*** migration.
 - [PhpArrayTable]: Removed PhpArrayTable class
 - [Navigation Builder] Calling the method `Pimcore\Navigation\Builder::getNavigation()` using extra arguments is
   removed. Instead, please pass the arguments as an associative array (eg.`getNavigation($args)`.For details, please see [#12310](https://github.com/pimcore/pimcore/issues/12310)
+- [Flysystem] Bumped `league/flysystem-bundle` minimum requirement to ^3.0 (which introduces `directoryExists()`,`has()` methods and fixes support for `directory_visibility` configuration option). NB: [Visibility](https://flysystem.thephpleague.com/docs/visibility/) permissions were specified as `0644` for files and `0755` for directories, are now using the more simplified `public/private` options which are translated respectively into `0600` and `0744`. 
```

Comment:
one last thing...if a user requires package for using third party adapters e.g. [aws s3](https://flysystem.thephpleague.com/v1/docs/adapter/aws-s3-v3/) then it should be upgraded to version 3 otherwise it will conflict on composer update. should we also describe it here?